DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/styles/Site.css

```diff
@@ -0,0 +1,22 @@
+﻿.glyphicon-lg{font-size:3em}
+.blockquote-box{border-right:5px solid #E6E6E6;margin-bottom:25px}
```

Comment:
this class name `blockquote-box`... where is it from? I hope - not from bootstrap. it describes the element purpose?